VotingApplication/VotingApplication.Web/Scripts/Controllers/Create/ConfirmStepController.js

```diff
@@ -17,6 +20,11 @@
         $scope.styleIsValid = styleIsValid;
         $scope.expiryIsValid = expiryIsValid;
         $scope.invitationsAreValid = invitationsAreValid;
+        $scope.formatUtcPollExpiry = formatUtcPollExpiry;
+        $scope.expiryDateIsInPast = expiryDateIsInPast;
+        $scope.createPoll = createPoll;
+
+        var startingDate = new Date();
```

Comment:
This should be using `moment`, not `Date()`, as it's not explicit whether we're in local or UTC.